Applications/Games/DC Universe Online/Online/script.json

```diff
@@ -4,9 +4,7 @@
     "compatibleOperatingSystems" : [
         "LINUX"
     ],
-    "testingOperatingSystems" : [
-        "LINUX"
-    ],
-    "free"          : true,
-    "requiresPatch" : false
+    "testingOperatingSystems" : [],
```

Comment:
Could you add macOS in testing? I do not see anything that would prevent the game from running under macos.

Reply:
Why not in stable?

Reply:
Do you have a mac to be sure it works ?

Reply:
No.

Reply:
So testing ^^.